jsonschema-core.xml

```diff
@@ -709,11 +714,14 @@
                 </t>
                 <section title="Loading a referenced schema">
                     <t>
-                        To differentiate between schemas in a vast ecosystem, schemas are
-                        identified by URI. As specified above, this does not necessarily mean
-                        anything is downloaded, but instead JSON Schema implementations SHOULD
-                        understand ahead of time which schemas they will be using, and the URIs that
-                        identify them.
+                        The use of URIs to identify remote schemas does not necessarily mean anything is downloaded,
+                        but instead JSON Schema implementations SHOULD understand ahead of time which schemas they will be using,
+                        and the URIs that identify them.
+                    </t>
+                    <t>
+                        When schemas are downloaded,
```

Comment:
I don't follow this paragraph.  Taking the dependent clause out I read "When schemas are downloaded... see Usage over HTTP" which is a bit unclear.

Could this be rephrased as something like:

"For use cases when it is desirable to download schemas, such as a hypermedia system where schemas are discovered at runtime, see <xref>Usage for Hypermedia</xref>"

Note I'm proposing linking to section 11 as a whole rather than 11.3, because the actual linking of a downloadable schema is covered in 11.1, and I think that is needed to fully understand the issue (along with 11.3).

Reply:
Oh, I totally missed that. Yes, I was looking for that section.